ArticleTemplates/assets/js/modules/premiumTaster.js

```diff
@@ -57,20 +57,25 @@ function init() {
 function indicatorHtml(taster) {
     if (taster !== "adFree" && taster !== "offline") return "";
     let text = ""
+    let accessibilityText = ""
     let svg = ""
     if (taster === "adFree") {
         text = "Experiencing the app ad free is a Premium feature. As a new user you can enjoy it for <strong>free for one week.</strong>"
+        accessibilityText = "Experiencing the app ad-free is a Premium feature. As a new user you can enjoy it for free for one week."
         svg = adFreeSvg
     }
 
     if (taster === "offline") {
         text = "Enhanced offline reading is a Premium feature. As a new user you can enjoy it for <strong>free for one week.</strong>"
+        accessibilityText = "Enhanced off-line reeding is a Premium feature. As a new user you can enjoy it for free for one week."
```

Comment:
`reeding` isn't a typo here.. otherwise VoiceOver reads it like Reading, England .. 😏

Reply:
Same with `off-line` and `ad-free` .. otherwise the intonation is broken too.

Reply:
That's really odd, surely reading is used loads more than Reading?
I wonder how Android pronounces it + would screenreader users understand _Reading_ if it's used elsewhere?

Reply:
That's bizarre. Is that VoiceOver? Sounds like a bug worth reporting?

Reply:
It is odd indeed.. I can't find much about if that's a typo or not .. but things like Live we write as Lyve for a11y in iOS. Who would you report the bug with though?